### PR TITLE
wow such colors

### PR DESCRIPTION
**Pull request by raybuhr**

### CODE REVIEW

File: notebooks/plot_some_color_palettes.ipynb [hunk 1]
```diff
@@ -0,0 +1,268 @@
+{
```
Comment (raybuhr) [inline notebooks/plot_some_color_palettes.ipynb:1]: <p>interesting citation. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lruhlen/ReviewNB_test/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (raybuhr) [inline notebooks/plot_some_color_palettes.ipynb:1]: <p>such pretty!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lruhlen/ReviewNB_test/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (lruhlen) [inline notebooks/plot_some_color_palettes.ipynb:1]: <p>Please clear outputs before committing!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lruhlen/ReviewNB_test/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (lruhlen) [inline notebooks/plot_some_color_palettes.ipynb:1]: <p>Can we add a Sesame Street color theme?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lruhlen/ReviewNB_test/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

**Pull request closed without merging**